pkg/cmd/codespace/create.go

```diff
@@ -38,6 +39,7 @@ func newCreateCmd(app *App) *cobra.Command {
 
 	createCmd.Flags().StringVarP(&opts.repo, "repo", "r", "", "repository name with owner: user/repo")
 	createCmd.Flags().StringVarP(&opts.branch, "branch", "b", "", "repository branch")
+	createCmd.Flags().StringVarP(&opts.location, "location", "l", "", "location (assigned by IP or user settings if not provided)")
```

Comment:
If we expect non-devs to want to use this flag, would we explain the format of values that this takes?

Reply:
Updated to enumerate the values we'd accept today in the description of this flag— as @josebalius called out we may need to come back and update this later though.